medical_physician/models/medical_physician_service.py

```diff
@@ -1,40 +1,47 @@
-# -*- coding: utf-8 -*-
-# © 2016 LasLabs Inc.
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
-from odoo import fields, models
-
-
-class MedicalPhysicianService(models.Model):
-    """
-    Services provided by the Physician on a specific medical center.
-
-    A physician could have "surgeries" on one center but only
-    "general consultation" in another center,
-    or the same service with different prices for each medical center.
-    That's the reason to link this to res.partner instead of
-    medical_physician.
-    """
-    _name = 'medical.physician.service'
-    _inherits = {'product.product': 'product_id', }
-    _description = 'Medical Physicians Services'
-
-    product_id = fields.Many2one(
-        string='Related Product',
-        help='Product related information for Appointment Type',
-        comodel_name='product.product',
-        required=True,
-        ondelete='restrict',
-    )
-    physician_id = fields.Many2one(
-        string='Physician',
-        help='The physician for the appointment',
-        comodel_name='medical.physician',
-        required=True,
-        index=True,
-        ondelete='cascade',
-    )
-    # service_duration = fields.Selection(
-    #     minutes, string='Duration',
-    #     help='Duration of the appointment in minutes',
-    # )
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class MedicalPhysicianService(models.Model):
+    """ Services provided by the Physician at a specific medical center.
+
+    A physician could have "surgeries" on one center but only
+    "general consultation" in another center, or the same service
+    with different prices at each medical center.
+    """
+
+    _name = 'medical.physician.service'
+    _inherits = {'product.product': 'product_id'}
+    _description = 'Medical Physician Services'
+
+    product_id = fields.Many2one(
+        string='Related Product',
+        help='Product related information for service type',
+        comodel_name='product.product',
+        required=True,
+        ondelete='restrict',
+    )
+    physician_id = fields.Many2one(
+        string='Physician',
+        help='The physician for the appointment',
+        comodel_name='medical.physician',
+        required=True,
+        index=True,
+        ondelete='cascade',
+    )
+    center_ids = fields.Many2many(
```

Comment:
I think it is good as it is because `relation` attribute is optional:

> The attributes relation, column1 and column2 are optional.

https://www.odoo.com/documentation/10.0/reference/orm.html#fields